man/iocage.8

```diff
@@ -0,0 +1,1809 @@
+.Dd April 20, 2017
```

Comment:
Instead of adding this back in, you should just revert the removal commit. It makes the history of the manpage file much cleaner. Now we just see a giant added block, hard to diagnose when/where things went wrong.